external/runtime/runtime.depproj

```diff
@@ -86,7 +89,7 @@
     <Copy SourceFiles="@(DotnetExe)"
           DestinationFolder="$(TestHostRootPath)"
           SkipUnchangedFiles="true"
-          UseHardlinksIfPossible="true" />
+          UseHardlinksIfPossible="$(UseHardlink)" />
```

Comment:
Should we also use  this property in the copy call above?

Reply:
I was thinking about it but it is not needed. The hardlink works, but confuses dotnet about what the location is. We do not have same issue with shared libraries. and it can save some space & processing time.